### PR TITLE
Portfolio setup and mobile first

### PR DESCRIPTION
**Pull request by Praveen1995-10-14**
"Template1 portfolio project"
-I add the index.html file to give the structure of my portfolio.
-I add a style.css file to give the styling of my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    </div>
+                </div>
+            </div>
+            <main>
+                <div class="bg_image">
+                    <img class="background" src="images/header/header.png" alt="background_img">
+                </div>
+                <div class="intro">
+                    <h2>I’m Praveen Glad to see you!</h2>
+                </div>
```
Comment (roshan-bajgain) [inline index.html:42]: - [ ] Kindly adjust the spacing between the title and content. please see the image below for reference.
- [ ] kindly make the title bolder as well to stick to the design
### Figma design
figma
### Your design
your

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Praveen1995-10-14, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
- [ ] please name the branch in a more descriptive way as the project is about portfolio setup so you can use `portfolio-setup` etc. Implement from next milestone please
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Praveen1995-10-14) [reply index.html:42]: But on inspect in figma it's shows h2 tag, that why I used h2.
But i haven't any issue I'll change it.
Thanks

Comment (Praveen1995-10-14) [reply index.html:42]: Okay I will do it.
Thanks for guiding me.🙏

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    <div class="network">
+                        <img src="images/header/tower.png" alt="network_img">
+                    </div>
+                    <div class="wifi">
+                        <img src="images/header/Wifi.png" alt="wifi_logo">
+                    </div>
+                    <div class="battery">
+                        <img src="images/header/Battery.png" alt="battery">
+                    </div>
+                </div>
```
[the remark this answers is not in the document]
Comment (Praveen1995-10-14) [reply index.html:28]: so, i have to remove these classes.

**Comment by Praveen1995-10-14** [comment]
hello roshan thanks for guiding me,
I have a confusion can you plz counter my question:-
so, in my toolbar I have to remove my all classes like time,battery,wifi
and network but there I only make a class name by them.
what to do I'm little confused

**Pull request merged**